README.md

```diff
@@ -1,2 +1,221 @@
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Catlog] <a name="about-project"></a>
+
+> In this project, we have created a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below. The data will be stored in JSON files but we also prepare a database with tables structure analogical to your program's class structure.
+
+- Create a main.rb file that will serve as your console app entry-point.
+  - Present the user with a list of options to perform.
+  - Let users choose an option.
+  - If needed, ask for parameters for the option.
+  - Have a way to quit the app.
+- The following options should be available:
+  - List all books
+  - List all music albums
+  - List all movies
+  - List of games
+  - List all genres (e.g 'Comedy', 'Thriller')
+  - List all labels (e.g. 'Gift', 'New')
+  - List all authors (e.g. 'Stephen King')
+  - List all sources (e.g. 'From a friend', 'Online shop')
+  - Add a book
+  - Add a music album
+  - Add a movie
+  - Add a game
+
+## Vedio Demo
+
+[Video](./VedioDemo.mp4)
+
+## 🛠 Built With <a name="built-with">Ruby</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+https://github.com/TracyMuso/Catalog
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
```

Comment:
[REQUIRED] Your README does not give correct instructions on how to run your app.